### PR TITLE
Added negative scenario test cases

### PR DESCRIPTION
**Pull request by Hariharayugan**

### CODE REVIEW

File: modules/idaithalam/src/main/java/io/virtualan/idaithalam/core/api/ParallelExecutor.java [hunk 1]
```diff
@@ -39,18 +40,22 @@ public Integer call() {
             && apiExecutorParam.getInputExcel() != null)
             && !Execution.EXECUTE.name().equalsIgnoreCase(apiExecutorParam.getExecution().name()))
          {
-          ExcelToCollectionGenerator.createCollection(apiExecutorParam);
+       flag=   ExcelToCollectionGenerator.createCollection(apiExecutorParam);
```
Comment (elans3) [inline modules/idaithalam/src/main/java/io/virtualan/idaithalam/core/api/ParallelExecutor.java:43]: any reason the flag was added?

File: modules/idaithalam/src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (elans3) [inline modules/idaithalam/src/main/resources/logback.xml:1]: Can we remove the logback.xml

File: modules/idaithalam/src/main/java/io/virtualan/idaithalam/core/api/ParallelExecutor.java (hunk 1 above)
Comment (Hariharayugan) [reply modules/idaithalam/src/main/java/io/virtualan/idaithalam/core/api/ParallelExecutor.java:43]: if ExcelCollectionGenerator is not able to generate the collection, due to some valid reason, we should validate the contract. That's why assigning  the return value to flag and verifying the flag value before procedding

File: modules/idaithalam/src/main/resources/logback.xml (hunk 2 above)
Comment (Hariharayugan) [reply modules/idaithalam/src/main/resources/logback.xml:1]: will do

**Pull request merged**